src/components/ChatBot/ChatFloatingModal.tsx

```diff
@@ -73,19 +75,46 @@ export default function ChatFloatingModal () {
 	const toggleChat = () => {
 		setIsOpen((prev) => !prev);
 	};
+
+	const { network } = useNetworkContext();
+	const grillData = {
+		[globalNework.CERE]: ['5145', '5139'],
+		[globalNework.KILT]: ['2035', '5144'],
+		[globalNework.KUSAMA]: ['2027', '5138'],
+		[globalNework.MOONBEAM]: ['2065', '5140'],
+		[globalNework.POLKADOT]: ['3638', '754']
+	};
 	const hasOpened = useRef(false);
 	useEffect(() => {
 		if (!isOpen) return;
 
 		if (!hasOpened.current) {
-			grill.init({ hub: { id: 'polka' }, theme: 'light' });
+
+			let filterArray: string[] = [];
+			if ((grillData as any)[network]) {
+				// eslint-disable-next-line @typescript-eslint/no-unused-vars
+				filterArray = (grillData as any)[network];
+			}
+			grill.init(
+				{
+					hub: { id: 'polkassembly' },
+					onWidgetCreated: (iframe) => {
+						const channelsQuery = new URLSearchParams();
+						const filterChannels = filterArray;
+						channelsQuery.set('channels', filterChannels.join(','));
+						iframe.src += `&${channelsQuery.toString()}`;
+						return iframe;
+					}
+				}
+			);
 		}
 		hasOpened.current = true;
-	}, [ isOpen ]);
+	// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [grillData,network]);
 
 	return (
 		<Container>
```

Comment:
use condition before the `Container` component and return null if `grillData[network]` is undefined,